libapicore/ApiServer.cpp

```diff
@@ -36,7 +37,7 @@ void ApiServer::start()
 		return;
 	}
 
-	cnote << "Api server listening for connections on port " + to_string(m_acceptor.local_endpoint().port());
+	cnote << "Api server listening for connections on port " + to_string(m_acceptor.local_endpoint().port()) << (m_password.empty() ? "" : " Authentication needed");
```

Comment:
Maybe add a `.` before ` Authentication needed`?

Reply:
It's just a sophistication ... that line will be quickly scrolled outside the boundaries of the screen.
Anyway ... ok.

Reply:
You can remove "for connections" part of the message.